.gitlab-ci.yml

```diff
@@ -124,3 +127,9 @@ publish:s3:
         s3://$S3_BUCKET_NAME/${RASPBIAN_NAME}/arm/${RASPBIAN_NAME}-mender-${MENDER_CLIENT_VERSION}.img.xz
     - aws s3api put-object-acl --acl public-read --bucket $S3_BUCKET_NAME
         --key ${RASPBIAN_NAME}/arm/${RASPBIAN_NAME}-mender-${MENDER_CLIENT_VERSION}.img.xz
+
+    - mender-artifact modify -n release-1 deploy/raspberrypi-${RASPBIAN_NAME}-mender-${MENDER_CLIENT_VERSION}.mender
```

Comment:
I think we should do the actual conversion in `convert` stage, pass it as a build artifact, and here only upload it.

Reply:
I thought about it, but this is tricky, because the tests test the name, and would fail if it was different. We also cannot rename during the mender-convert stage itself, because then they will be the same, which we're trying to avoid.

The renaming is specifically for our distribution and our demo, so I think it's alright to do it here. It's not how mender-convert should work in general.

Reply:
Ok. Agreed.